daemon/cmd/daemon_main.go

```diff
@@ -1175,6 +1175,11 @@ func initEnv(cmd *cobra.Command) {
 		}
 	}
 
+	if option.Config.EnableBandwidthManager && option.Config.EnableIPSec {
+		log.Warning("IPSec cannot be used with the bandwidth manager. Disabling the latter.")
```

Comment:
Just a small nit, to avoid confusing for non-native speakers, maybe should be more clear to the user: "IPSec cannot be used with the bandwidth manager. Disabling the bandwidth manager."

Reply:
Sure. I've also reverted the first sentence to avoid making the repetition too obvious.